---
string direction  # The current rollerbrush direction (up/down)
